scripts/backend/backend.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+export DEBIAN_FRONTEND=noninteractive
+apt-get update -y
+
+# Step 1. Install swtpm requirements
+apt install -y autoconf-archive libcmocka0 libcmocka-dev procps iproute2 build-essential git pkg-config gcc libtool automake libssl-dev uthash-dev autoconf doxygen libjson-c-dev libini-config-dev libcurl4-openssl-dev libltdl-dev expect python3-yaml libglib2.0-dev dbus  libtasn1-6-dev libjson-glib-dev gawk socat libseccomp-dev uuid-dev udev libusb-1.0-0-dev libgnutls28-dev gnutls-bin
+
+# Step 2. Install acs requirements
+apt install -y libjson-c4 libjson-c-dev mariadb-server libmysqlclient-dev git make gcc wget && rm -rf /var/lib/apt/lists
+
+# Step 3. download submoudle
+CURRENT_DIR=$(pwd)
```

Comment:
Please use `WORKDIR` to replace the `CURRENT_DIR`. And you can remove all `CURRENT_DIR` in this Dockerfile